py/server/deephaven/table.py

```diff
@@ -2084,6 +2084,28 @@ def slice(self, start: int, stop: int) -> Table:
         except Exception as e:
             raise DHError(e, "table slice operation failed.") from e
 
+    def slice_pct(self, start_pct: float, end_pct: float) -> Table:
+        """Extracts a subset of a table by row percentages.
+
+        Returns a subset of table in the range [startPercentInclusive * sizeOfTable , endPercentExclusive * sizeOfTable).
```

Comment:
Update the docs based on the javadoc comments to be precise about what the operation is doing.